etc/docker/elyra/Dockerfile

```diff
@@ -16,24 +16,33 @@
 #
 
 # Ubuntu 18.04 LTS - Bionic
-FROM jupyterhub/k8s-singleuser-sample:0.8.2
+FROM jupyterhub/k8s-singleuser-sample:0.9.0
 
-# pip packages
-RUN pip install --upgrade pip
-RUN pip install setuptools --ignore-installed --upgrade
+USER root
 
-ENV NBGITPULLER_APP=lab
+ADD start-elyra.sh /usr/local/bin/start-elyra.sh
 
-# Install Lab which is enabled by default
-RUN pip install --upgrade pip jupyterlab notebook
+RUN chmod ugo+x /usr/local/bin/start-elyra.sh && \
+    apt-get update && apt-get install -y build-essential curl && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list && \
+    apt-get update && apt-get install -y yarn && \
+    rm -rf /var/lib/apt/lists/*
 
-RUN mkdir -p /home/jovyan/.local/share/jupyter/metadata/runtimes
+USER $NB_USER
+#USER jovyan
```

Comment:
Can we remove this commented out command?  If its purpose is to indicate that NB_USER will likely be jovyan, then we should add a more formal comment, but I don't think that is necessary.

Reply:
sorry, i forgot to push the latest changes, including this one.